### PR TITLE
feat(esx_ambulancejob/server/main): Added state bags

### PR DESCRIPTION
**Pull request by Thekuca**
Added a isDead state bag. Will be very useful in the future in other scripts so we don't have to use on player death and on player spawn handlers we can just check by LocalPlayer.state.isDead

### CODE REVIEW

**Comment by PsychoShock** [comment]
You should change to be able to make check on client too.

File: server/main.lua [hunk 1]
```diff
@@ -65,6 +68,8 @@ AddEventHandler('esx:onPlayerDeath', function(data)
 	local source = source
 	deadPlayers[source] = 'dead'
 	local Ambulance = ESX.GetExtendedPlayers("job", "ambulance")
+	local plyState = Player(source).state
+	plyState.isDead = true
```
Comment (Gellipapa) [inline server/main.lua:72]: I think you should make function and call everywhere and pass source than parameter.

**Review by Gellipapa — CHANGES_REQUESTED** [review]

**Comment by Gellipapa** [comment]
> You should change to be able to make check on client too. This code is only for server state.

@PsychoShock  Thank you for tested.

**Comment by Gellipapa** [comment]
@Thekuca Hi! Please refactor code and make works both side.

**Review by Gellipapa — COMMENTED** [review]
Thank you, looks cool.

**Pull request merged**